src/main/java/uk/gov/justice/probation/courtcaseservice/jpa/entity/OffenderAliasEntity.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.justice.probation.courtcaseservice.jpa.entity;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDate;
+
+@Data
+@Builder
+@NoArgsConstructor(access = AccessLevel.PRIVATE, force = true)
+@AllArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class OffenderAliasEntity {
+
+    private final LocalDate dateOfBirth;
+    private final String firstName;
+    private final String surname;
+    private final String gender;
```

Comment:
Looking at the API [here](https://probation-offender-search-dev.hmpps.service.justice.gov.uk/swagger-ui/index.html#/offender-match/match) it looks like an array of `middleNames` can be returned as well so we should add those...

It would be good to reuse the `NamePropertiesEntity` in fact... but that stores it as three forenames 🙄 
```
    @Type(type = "jsonb")
    @Column(columnDefinition = "jsonb", name = "NAME", nullable = false)
    private final NamePropertiesEntity name;
    ...
    private String forename1;
    private String forename2;
    private String forename3;
```

That's annoying... We'll just have to get the matcher to mung any excess middle names down into forename3 when it posts.  

EDIT: Don't do this ☝️  See this 👇

Reply:
Urgh... but then again this whole class is getting the JSONB treatment... so we're kinda making a rod for our own back if we do that.  

Ok change of plan - Keep it as we receive it from offender search, and store it exactly as specified with a List of `middleNames`. The matches tables are effectively a local cache of offender search results anyway, it's not data that we 'own' as such so I think perhaps our interfaces should default to whatever offender search produces.  I'll add an ADR to this effect so we've got some guiding principles for future questions around this.

Reply:
addressed